test/LibraryManager.IntegrationTest/LibmanTestsUtility.cs

```diff
@@ -8,6 +9,142 @@ namespace Microsoft.Web.LibraryManager.IntegrationTest
 {
     public class LibmanTestsUtility
     {
+        private static HashSet<string> GetSubDirectoriesAndFiles(string currentWorkingDirectory, bool caseInsensitive)
```

Comment:
>private static HashSet<string> GetSubDirectoriesAndFiles(string currentWorkingDirectory, bool caseInsensitive) [](start = 8, length = 110)

nit: I would probably implement this like the following

        private static HashSet<string> GetSubDirectoriesAndFiles(string currentWorkingDirectory, bool caseInsensitive)
        {
            StringComparer comparer = caseInsensitive ? StringComparer.OrdinalIgnoreCase : StringComparer.Ordinal;
            IEnumerable<string> subItems = Directory.EnumerateFileSystemEntries(currentWorkingDirectory, "*", SearchOption.AllDirectories);

            return new HashSet(subItems, comparer);
        }

But it doesn't really bother me the other way.

Reply:
I see, thanks for pointing out

---
In reply to: [195502551](https://github.com/aspnet/LibraryManager/pull/113#discussion_r195502551) [](ancestors = 195502551)